projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.ts

```diff
@@ -135,6 +159,46 @@ export class TableBasicComponent implements OnInit, OnDestroy {
         return isLast && (hasMore || this.loadingChildRows[rowIndex] || this.failedChildRows[rowIndex]);
     }
 
+    getRowspanIndexes(row): number[] {
+        return [...Array(this.tableService.getMaxRowspanInColumn(row)).keys()].filter(Boolean);
+    }
+
+    private setHoverForRowspan() {
+        const rowElements = this.elementRef.nativeElement.querySelectorAll('tr[data-row-idx]');
+        const events = ['mouseenter', 'mouseleave'];
+        from(events)
+            .pipe(
+                takeUntil(this.onDestroy$),
+                mergeMap(event => fromEvent(rowElements, event)),
+                filter((event: MouseEvent) => {
+                    return (
+                        (event.type === 'mouseenter' && !(event.target as HTMLElement).classList.contains(ROW_HOVERED_CLASS_NAME)) ||
```

Comment:
filter for "mouseenter" in case target don't have a "hover" class

Reply:
please confirm that takeUntil should be at the beginig and not at the end when using mergeMap opertator